azdev/operations/tests/__init__.py

```diff
@@ -103,13 +103,16 @@ def _find_test(index, name):
 
 
 def _extract_module_name(path):
-    mod_name_regex = re.compile(r'azure-cli-([^/\\]+)[/\\]azure[/\\]cli')
-    ext_name_regex = re.compile(r'.*(azext_[^/\\]+).*')
+    core_name_regex = re.compile(r'azure-cli-(?P<name>[^/\\]+)[/\\]azure[/\\]cli')
+    mod_name_regex = re.compile(r'azure-cli[/\\]azure[/\\]cli[/\\]command_modules[/\\](?P<name>[^/\\]+)')
```

Comment:
When you pull out the paths, I might add these regexp definitions near them, so that anybody modifying one or the other in the future sees immediately that both will need to be updated.